frontend/packages/ceph-storage-plugin/src/models.ts

```diff
@@ -12,3 +12,16 @@ export const CephClusterModel: K8sKind = {
   id: 'cephcluster',
   crd: true,
 };
+
+export const OCSModel: K8sKind = {
```

Comment:
Nit: I'd suggest adding at least `Service` to the name of the model, like `OCSServiceModel`.